<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="questionBase_load">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Question Base</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userProfile_load">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load User Profile</string>
    </property>
   </widget>
   <widget class="QComboBox" name="questionBase_box">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="userProfile_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="userProfile_new">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userProfile_delete">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questionBase_new">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New Question Base</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questionBase_delete">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Question Base</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionbase_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>175</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Question Base:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="question_write">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write New Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="question_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Questions</string>
    </property>
   </widget>
   <widget class="QListWidget" name="categories_list">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="categories_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>To only quiz yourself on specific categories, make selections here. If &lt;ALL&gt; is selected, no matter what other categories were selected, ALL will be used.</string>
    </property>
    <property name="text">
     <string>Select Question Categories</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_quiz">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_stats">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>See User Stats</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numquestions">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Please insert a number. Any non-numeric addition will default to ALL.</string>
    </property>
    <property name="text">
     <string>Specify number of questions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questions_download">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Questions</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGet_Stats">
   <property name="text">
    <string>Get Stats</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
